plugin.go

```diff
@@ -102,7 +103,7 @@ func installCaCert(cacert string) *exec.Cmd {
 
 func exportSecrets(secrets map[string]string) {
 	for k, v := range secrets {
-		os.Setenv(fmt.Sprintf("%s", k), fmt.Sprintf("%s", os.Getenv(v)))
+             os.Setenv(k, v)
```

Comment:
I think removing the `fmt.Srptinf` is OK, but seems we accidentally got rid of the `os.Getenv(v)` part as well?  I think we want to keep that.